package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -148,21 +257,31 @@ def _single_frame(self) -> None:
         self._res_array[self._frame_index, :] = dist
 
     def _conclude(self) -> None:
-        self.results['pair_distances'] = self._res_array
+        self.results['distances'] = self._res_array
```

Comment:
If you want to change the name then you need to
1. deprecate pair_distances (add entry to CHANGELOG and `.. deprecated` to the entry)
2. keep an entry `self.results['pair_distances'] = self.results['distances']` until it's removed in 3.0.0

The entries in `results` are part of the API so they fall under SemVer.

Reply:
OK, I'll add a deprecation notice and add that to the docs and change log

Reply:
It's done, I added it to the docs.